pages/draw.md

```diff
@@ -262,6 +262,37 @@ poi-icons:
             sprite_default: generic
 ```
 
+####`collide`
+Optional _boolean_. Defaults to `true`.
```

Comment:
My expectation of `collide: false` is that the object doesn't participate in any collisions (e.g. isn't added to the collision set at all). We discussed that `false` would have no collisions, and in the future we could extend to allow more specific partial collisions, such as by specifying the names of other layers to collide with.